### ISSUE TITLE
Escape Characters

### ISSUE THREAD

**Issue by hayesgm** [danielberkompas/ex_twiml#17]
I just created the following TwilML:

```elixir
msg = "Hello <> Goodbye"

twiml do
      say msg
end
```

and that generated the following XML:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Response>
    <Say>Hello <> Goodbye</Say>
</Response>
```

That's clearly not the expected behavior, specifically, we should expect:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Response>
<![CDATA[Hello <> Goodbye]]>
</Response>
```

Should we either 1) add escaping, or 2) add an actual XML generation library? In either case, we need to handle escaping of special characters in XML since this could be a huge security flaw due to injection if you have any dynamically generated data in your TwilML templates.

**Comment by danielberkompas** [comment]
What I would like to do is have the `twiml` macro build up a map instead of a binary, and finally pass it to an XML encoder. I don't have time to do this now, but will keep it in mind.

**Comment by danielberkompas** [comment]
This should be fixed by #19.
